requirements/edx/github.in

```diff
@@ -61,6 +61,10 @@ git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752
 -e git+https://github.com/dementrock/pystache_custom.git@776973740bdaad83a3b029f96e415a7d1e8bec2f#egg=pystache_custom-dev
 -e git+https://github.com/jazkarta/edx-jsme.git@690dbf75441fa91c7c4899df0b83d77f7deb5458#egg=edx-jsme
 
+# Forked to get Django 2.2 support from unreleased master branch from social-app-django repo.
+# This can be removed once an official social-auth-app-django Pypi release with Django 2.2 support is available in the future.
+-e git+https://github.com/python-social-auth/social-app-django.git@c00d23c2b45c3317bd35b15ad1b959338689cef8#egg=social_django
```

Comment:
Shouldn't the egg name be `social-auth-app-django` to match the package name?

Reply:
The egg name is normally the core folder name from the package. In current case, it is `social_django` as you can see on https://github.com/python-social-auth/social-app-django.

Reply:
That's not what the pip documentation suggests: https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support . I'm not sure it technically matters in this case since we don't have any other dependencies that depend on this one, just want to make sure we understand how this works so we don't make future mistakes involving it.

Reply:
Updated the egg name as per the documentation.